taskcluster/app_services_taskgraph/loader/build_config.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Same here, maybe we should have a quick intro in each file to describe what they do?

Reply:
This loader is a special one. It's only used by https://github.com/mozilla/application-services/pull/2820/files/39b0c15c030f2aacbf0177b858143b7f8696a94d#diff-037116387fd4f688ffd8855e6f3969d5R5.

It generates a new build task for every component defined in `.buildconfig-android.yml`. It does nothing more.

Reply:
Added an explanatory comment at the beginning of the file.